### ISSUE TITLE
Estimating Lambda values

### ISSUE THREAD

**Issue by DMario93** [wagner-group/active-learning#4]
Dear authors, 

I need a clarification on the lambda values you use for scaling your end-to-end loss. According to the paper, this is chosen such that "the average of the two terms Lhc and λLce have a similar mean, so the overall loss is not dominated/overwhelmed by just one term." This suggests that the lambda value is either updated dynamically after each training epoch (or after a certain number of batches) or estimated approximately beforehand. 

From your code it seems that you somehow estimated it to be 100 beforehand. Could you help me figure out how to compute this value? 

Thank you for your help!

**Comment by surrealyz** [comment]
Dear Mario,

You could print out the two loss values without setting the lambda first. Then, you could see how to scale the loss values accordingly so the two terms have similar averages over the gradient descent steps.

Best,
Yizheng
